.github/workflows/reusable-add-reviewers-to-pr.yml

```diff
@@ -31,11 +31,13 @@ jobs:
       pull-requests: write
     steps:
       - id: get-members
+        env:
+          GH_TOKEN: ${{ secrets.token }}
         run: |
           DATA=$(gh api \
             -H "Accept: application/vnd.github+json" \
             -H "X-GitHub-Api-Version: 2022-11-28" \
-            /orgs/${{ github.repository_owner }}/teams/${{ inputs.team-slug }}/members?role=maintainer&per_page=100 \
+            /orgs/${{ github.repository_owner }}/teams/${{ inputs.team-slug }}/members&role=maintainer?per_page=100 \
```

Comment:
This seems wrong, query string is `?aaaa=1&bbbb=2`

Reply:
i just removed them ... its unlikely that we will have teams with 100+ members anyways and all of them will be maintainers.

It did work on my local console though, so not sure why that did not work the other way around 🤷🏼